<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find turtle2_openai_ros_example)/config/turtlebot2_openai_qlearn_params_v2.yaml" />
    <!-- Launch the training system -->
    <node pkg="turtle2_openai_ros_example" name="test_turtlebot2_maze_dqn" type="test_deepq.py" output="screen"/>
</launch>
